### DISCUSSION TITLE
Trouble getting catppuccin loading correctly during fresh install - showing hostname instead of current dir

### DISCUSSION THREAD

**Discussion by rosco-dev** [catppuccin/tmux#400] [Help] [answered]
I am setting up a WSL environment running Ubuntu 24.04.1 LTS and catppuccin doesn't seem to be loading correctly (at least when comparing to my Ubuntu 20.04 environment). See screenshot:

![image](https://github.com/user-attachments/assets/b31ac993-b798-4ad7-b389-6a8154d5aa84)

I've previously used catppuccin with minimal configuration, basically using it with no configuration changes.

Differences I noticed:
- [0] to the far left
- Window name is my hostname instead of current directory
- Hostname repeated on the right
- Current application name not shown to the right
- Sessions not shown to the right

I copied a bunch of settings from the "Recommended Default Configuration" section of the readme, which addresses most of these, however I haven't worked out how to get the current directory to show for the window name.

What I am hoping to find out is if my catppuccin is not loading correctly for me, or if the out of the box look of catppuccin has changed and I need to make these config changes myself (and if so, what do I need to set to set my current working dir to the window name).

Thanks in advance!

More details:
Running on WSL using the Windows Terminal
tmux 3.5a
I have tried to install following both the manual (recommended) way and via TPM, same result for both
tmux.conf:

```
# Set the default terminal terminfo
set -g default-terminal "tmux-256color"
# True colour support
set -as terminal-overrides ",xterm*:Tc"

# List of plugins
set -g @plugin 'tmux-plugins/tpm'
set -g @plugin 'tmux-plugins/tmux-sensible'
set -g @plugin 'catppuccin/tmux'

# Workaround. Prevent appearing 0;10;1c when tmux start
set -sg escape-time 1
# Start windows and panes at 1, not 0
set -g base-index 1
setw -g pane-base-index 1

set -g mouse

# Initialize TMUX plugin manager (keep this line at the very bottom of tmux.conf)
run '~/.tmux/plugins/tpm/tpm'
```

**Comment by rosco-dev** [comment]
Right after posting this I stumbled upon another discussion https://github.com/catppuccin/tmux/discussions/386 which links to a migration guide. Pinning v0.3.0 fixed my issue, so this explains why i was seeing what I was seeing, curious if there is any recommended config to set to get the same look in v0.3.0 for latest, or is this no longer possible now?

Thanks

**Comment by kjnsn** [comment, marked as answer]
Everything is working exactly as intended. If you want your current working directory in the window name then you can instruct your shell to do that, or you can use a format option that tmux provides (see the man page)

**Reply by kjnsn** [reply]
So this is an issue with your setup and your shell is doing something weird

**Comment by kjnsn** [comment]
If you want something that just does everything for you, try tmux powerline.

**Comment by rosco-dev** [comment]
Thanks @kjnsn
